include/rviz_visual_tools/remote_reciever.h

```diff
@@ -36,8 +36,7 @@
    Desc:   Object for wrapping remote control functionality
 */
 
-#ifndef RVIZ_VISUAL_TOOLS_REMOTE_RECIEVER_H
-#define RVIZ_VISUAL_TOOLS_REMOTE_RECIEVER_H
```

Comment:
Oh man, this file was misspelled on creation :(

fyi in case you haven't already discovered, bitchin unix has a "gitspell" function which tries to spell check comments. It won't catch misspellings in file names or underscore delimited strings though. Maybe one day.